### DISCUSSION TITLE
Facebook和Twitter兩大社群媒體巨頭，生存期限快到了嗎？

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#860] [General]
臉書母公司Meta在執行長祖克柏（Mark Zuckerberg）的帶領下，大力推動、進軍元宇宙，但日前Meta宣布大舉裁撤13%、1.1萬名的人員，祖克柏也坦承自己過於樂觀看待市場預期，以致人員過剩。然而，稍早有外媒爆料祖克柏將於明年辭去Meta執行長職位，引起市場關注。

近幾年因疫情影響，為避免過度接觸而染疫，多數人選擇在家中使用社群媒體，致使科技業營收增長，facebook和twitter乘介了這項商機，在媒體中置入了許多廣告，廣告收入儼然成為了社群媒體的生存方式。Meta也決定要進軍新市場--元宇宙，然而，投入了大量的人力以及成本並沒有得到相對應有的回饋，且新興市場Tiktok崛起(以下稱抖音)，多數年輕人選擇使用抖音，減少了臉書及推特的使用，致使Meta必須要裁掉多數員工，但我覺得，以科技日新月異的腳步來看，或許一個社群軟體能有如此長的生命週期已很厲害了，只期待Meta未來能夠順利發展出更加厲害的社群軟體來回血。

https://www.thenewslens.com/article/176978

**Comment by bonny123865** [comment]
社群平台也是會隨著流行而變動，而且這一次裁員的舉動感覺對於社會或是經濟會有很大的影響。

**Comment by Pietra1226** [comment]
元宇宙最近熱度確實降低了不少，可能難的地方是將整個世界搬入，當初他們確實過於樂觀畫了大餅。

**Comment by Kotori0501** [comment]
社群媒體會隨著不同年齡層的流行改變，而有許多條件可以實現，但現在年輕人愛用抖音！！我還是比較習慣用fb和twitter耶！

**Comment by Chiway1229** [comment]
目前推特因為elon的關係，已經有點焦頭爛額了
傳統社群媒體的Youtube近年也因為各種政策與營利風波而持續有新聞
說不定這時出現一個更強大的軟體就能取代舊時代媒體也說不定呢

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
各個社群煤體都有一定的生命週期 這次兩大社群媒體的興衰也是無可必免的

**Comment by ching020721** [comment]
社群平台的興盛衰敗在所難免，小時候經常使用的平台隨著世代交替改變也是好事，市場上有流動才能刺激進步，像蝦皮購物平台之後可能也會被其他新興購物平台取代。

**Comment by peggy0329** [comment]
我認為祖克柏大量投入資金於元宇宙的原因可能也是看到臉書不能在單單只靠社群存活，臉書的社群媒體的生命週期可能已經開始進入衰退期，面對未來不確定的變遷，他可能在為數位轉型進行佈局，然而許多大公司大量裁員其實是聯準會持續升息帶來的必要結果，為了提高目前過低的失業率，已趨緩通膨。以我來說，還是習慣從臉書找尋一些重要的資訊，希望臉書除了透過轉型後、多角化經營，拜託也讓這個臉書這個大量資訊的管道不要脈絡！

**Comment by HuangHsinTzu** [comment]
我觀察一個公司的規模越來越大之後
對自己公司所做的決定越來越有信心
野心也來越來越大
利益導向的情形也愈來愈明顯
但那些大公司的CEO也是人，不是神
總有看錯商機的時候
所以這些巨頭公司偶爾有裁員的狀況是難免的
做為普通老百姓的我們可以參考這些巨頭公司的投資方向反應現今趨勢及預測未來趨勢
期待這些巨頭公司有更驚為天人的願景
也期待有更有創意的小公司崛起!

**Comment by StevenWanglolz** [comment]
我認為Elon Musk的行為有點讓大家看不懂? 但他是世界首富嘛，所以不懂也是正常的。不過身為一個Tik Tok用戶，我真的認為它比其他短影片app都來的有娛樂性。

**Comment by williamyen0623** [comment]
身為facebook最初的使用者，真心希望祖克柏能帶領meta再次掀起一波熱潮

**Comment by manipulateed** [comment]
一時的潮流造成對於未來過於樂觀的判斷，這不僅存在這件事情上面，投資標的或是一件事情的未來都需要慢慢地去觀察，而非一蹴可及的

**Comment by CPCpyt** [comment]
社群平台是人們共同意象的展現，
誰最能提供最符合人的需求就能夠搶占最多的市場，取得成功。

**Comment by ruthchen1020** [comment]
還是希望他們可以長壽，畢竟很多回憶都在上面，即使很少人用了可以還是開著伺服器嗎。
